src/hooks/useOAuthFlow.tsx

```diff
@@ -123,13 +125,14 @@ export const OAuthContextProvider = ({
       try {
         const result = await authorize(authConfig);
         await storeAuthResult(result);
+        _sdkTracker.track('Login', { usedInvite: !!inviteId && !!evc });
```

Comment:
I feel like this might read better if we changed the instance to be `_sdkEvents` it would also help avoid overloading the term `tracker`.

Reply:
Something like `Events.track` would read nicely.

Reply:
There are a few event emitters, I think I'll do `_sdkAnalyticsEvent` to make clear 🙏